editor-extensions/vscode/server/src/server.ts

```diff
@@ -315,11 +339,21 @@ async function validateWithCompiler(textDocumentUri: string): Promise<void> {
 
           let cwd = path.dirname(filename);
 
-          let result_json_buffer = childProcess.execFileSync(
-            cliPath,
-            ["lsp", filename],
-            { input: text, cwd }
-          );
+          let run_path = cliPath;
+          let args = [];
+
+          // check every run in case it's added or removed
```

Comment:
How slow does this make stuff?

Reply:
I could see us setting up a watcher on that file and toggling a flag if it changes instead of running an `exists` call on each run.

Reply:
Good question.  I figured compared to a compile or format, not much.  I did make it check on launch initially, but then I figured it would be confusing it you started vscode, then created a script and vscode ignored it until the next run.

Reply:
I'll take a look at the watcher option before I merge this

Reply:
chokidar watcher added